### ISSUE TITLE
Can i record a canvas that is a still image?

### ISSUE THREAD

**Issue by johnlandish** [SMUsamaShah/CanvasRecorder#4]
I have a small animating canvas on top of a larger jpg image canvas. Can i record them both as one? basically can i record a non moving canvas image or does it have to be animating for the recording to work

**Comment by SMUsamaShah** [comment]
No. currently it can only take one canvas as a reference and record it.

**Comment by johnlandish** [comment]
What if I put two canvases inside one canvas? Will that work?

**Comment by SMUsamaShah** [comment]
If two canvases are being drawn on to one canvas, then it sure will.
